### DISCUSSION TITLE
关于sevi第二阶段大作业的问题

### DISCUSSION THREAD

**Discussion by fuxuze** [physics-data/faq#168] [Q&A] [answered]
请问sevi第二阶段大作业一个电子的成像结果是高斯亮斑吗？如果是的话，是形如第一阶段大作业level1那样的亮斑还是level2那样的？每一个 测试集/训练集 内是否每个电子的成像函数对应的参数（展宽和A)是固定的吗？

**Comment by jsw2000** [comment]
提问2：请问sevi大作业测试集中图都是垂直方向为z，水平方向为x吗，具体坐标系如何确定？这对使用第二类数据集很重要。

**Reply by littlewu2508** [reply]
是的，垂直方向为z，水平方向为x

**Comment by littlewu2508** [comment, marked as answer]
> 
> 
> 请问sevi第二阶段大作业一个电子的成像结果是高斯亮斑吗？如果是的话，是形如第一阶段大作业level1那样的亮斑还是level2那样的？每一个 测试集/训练集 内是否每个电子的成像函数对应的参数（展宽和A)是固定的吗？

类似 level2，都是高斯型亮斑。见网路学堂最新更新的题面文件
